docs/src/cli/app-updates.md

```
@@ -0,0 +1,218 @@
+      "package": "myapp",
+      "command": "self-update"
+    }
+  }
+}
+```
+
+That is enough for the configuration side. Once you add the lookup call, a run
+that finds a newer `myapp` records it, and the next run prints two lines to
+stderr:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)